app/(editor)/editor/page.tsx

```diff
@@ -185,9 +65,13 @@ export default async function Page({ searchParams }: PageProps) {
 
   // Get data out of the API.
   const { data } = await getStoryData(searchParams);
-  let storyList;
-  if (slug === 'stories' || slug.includes('stories/list/')) {
-    storyList = await getStoryList(searchParams);
+
+  // An extra container for passing along additional fetched data.
+  let extra = {};
+
+  // Get additional data for those stories that need it.
+  if (data?.story?.content?.component === 'sbStoryFilterPage') {
```

Comment:
I changed the logic here to match on the page type instead of the path so that we can have portability in the Storyblok UI.

Reply:
Yeah I like your way better 👍🏼